build.gradle.kts

```diff
@@ -54,7 +54,7 @@ apiValidation {
 }
 
 // Semantic versioning for release version
-val versionName = "1.1.0"
+val versionName = "2.0.0-rc1"
```

Comment:
Do we need this if we use git to check tag anyway?

Reply:
This is for docs generation. The tag is created after this PR.
I haven't created this flow but I can look into it some time.